### DISCUSSION TITLE
resume a web game without using dashboard

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#18] [Q&A] [answered]
The buttons are not clickable after pausing, and requires users to go to dashboard to resume a game.  This may be confusing or needs to be documented. There also were these errors in console, probably unrelated but FYI

caught (in promise) Error: Could not establish connection. Receiving end does not exist.
dashboard:1 Uncaught (in promise) Error: Could not establish connection. Receiving end does not exist.

![Screen Shot 2023-04-24 at 6 39 48 PM](https://user-images.githubusercontent.com/4314771/234153670-798ee85d-b322-4402-b630-fc7b63617fbe.png)
[eagerwords-test.com-1682386661950.log](https://github.com/azadbolour/eagerwords-community/files/11315706/eagerwords-test.com-1682386661950.log)
[eagerwords-test.com-1682386677152.log](https://github.com/azadbolour/eagerwords-community/files/11315707/eagerwords-test.com-1682386677152.log)

**Comment by azadbolour** [comment, marked as answer]
Currently after a pause, it is necessary to go back to the dashboard, and
resume from there. The design was motivated by the need to just stop the
game and continue another day. Then resuming the game needs a protocol to
ask the opponent to join similar to starting a new game.

This state of affairs is not ideal for short pauses in which both sides
would simply stay on the game page and resume after a short interruption.
That is a known issue . But fixing it has not made it to the road map yet.

Azad
